website/docs/language/functions/cidrnetmask.mdx

```diff
@@ -20,7 +20,7 @@ cidrnetmask(prefix)
 The result is a subnet address formatted in the conventional dotted-decimal
 IPv4 address syntax, as expected by some software.
 
-CIDR notation is the only valid notation for IPv6 addresses, so `cidrnetmask`
+Because CIDR notation is the only valid notation for IPv6 addresses, `cidrnetmask`
```

Comment:
Hi! 👋  I agree this sentence could use further clarification, how would you feel about re-phrasing this to be even more explicit: 
```suggestion
 The `cidrnetmask` function only accepts IPv4 addresses in CIDR notation. If you use an IPv6 address, `cidrnetmask` returns an error. 
```